openapi/goto.meeting/Ballerina.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+org = "ballerinax"
+name = "goto.meeting"
```

Comment:
I think `gotomeeting` should be a single word. can you update the module folder name as well?

Reply:
But GoTo provides four APIs [GoToMeeting](https://developer.goto.com/GoToMeetingV1), [GoToWebinar](https://developer.goto.com/GoToWebinarV2), [GoToTraining](https://developer.goto.com/GoToTrainingV1) and [GoToConnect](https://developer.goto.com/GoToConnect). Multiple APIs from a vendor.

Reply:
It's a bit wired to have a dot separation here. Let's have names like `gotomeeting` `gotowebinar` `gototraining` `gotoconnect`

Reply:
Ack. Fixed in https://github.com/ballerina-platform/ballerinax-openapi-connectors/commit/fa14730cbafb911593c56574c6a8d825030f2546